nireports/reportlets/xca.py

```diff
@@ -237,6 +237,9 @@ def plot_melodic_components(
     )
     fig.clf()
 
+    if compress:
+        svg_compress(out_file, compress=compress)
```

Comment:
`svg_compress` expects the file contents, this is just passing in the filename, something like this should fix it.

```suggestion
        svg_compress(Path(out_file).read_text(), compress=compress)
```